fake_spectra/spectra.py

```diff
@@ -80,7 +80,7 @@ class Spectra:
                      kernel (a good back up for large Arepo simulations) "quintic" for a quintic SPh kernel as used in modern SPH
                      and "cubic" or "sph" for an old-school cubic SPH kernel.
     """
-    def __init__(self,num, base,cofm, axis, res=1., cdir=None, savefile="spectra.hdf5", savedir=None, reload_file=False, snr = 0., spec_res = 0,load_halo=False, units=None, sf_neutral=True,quiet=False, load_snapshot=True, gasprop=None, gasprop_args=None, kernel=None):
+    def __init__(self,num, base,cofm, axis, res=1., cdir=None, savefile="spectra.hdf5", savedir=None, reload_file=False, snr = [0.], CE=[0.],spec_res = 0,load_halo=False, units=None, sf_neutral=True,quiet=False, load_snapshot=True, gasprop=None, gasprop_args=None, kernel=None):
```

Comment:
Please don't use lists, arrays or other mutable datatypes as python default arguments, because python implements default arguments as simple global variables, any edits to the default argument can change the initial state. For example something like this:
CE += [1,]
would mean that future calls to init() have the default argument CE = [0,1] which is usually not what you mean. The standard thing to do is to use CE = None and set it to the list value in the function itself: "if CE is None: CE = [0,]"